Rock.Rest/Controllers/PeopleController.Partial.cs

```diff
@@ -1320,7 +1320,7 @@ public class PersonSearchResult
     }
 
     /// <summary>
-    /// 
+    ///

```

Comment:
You probably could dump changes to this file and any other files that just have whitespace changes in them. There's no need to include them here.